Management/src/ActuatorWeb/README.md

```diff
@@ -0,0 +1,201 @@
+﻿# Steeltoe Management Sample - Actuators, Administrative Tasks, Metrics and Tracing
+
+ActuatorWeb and ActuatorAPI form an ASP.NET Core-powered sample application that demonstrates how to use several
+Steeltoe libraries on their own and with additional tools such
+as [Tanzu Apps Manager](https://techdocs.broadcom.com/us/en/vmware-tanzu/platform/tanzu-platform-for-cloud-foundry/10-0/tpcf/console-index.html)
+and [Spring Boot Admin](https://docs.spring-boot-admin.com/) used for managing and monitoring applications.
+
+These samples also illustrate how to have application metrics captured and exported to
+the [Metrics Registrar](https://techdocs.broadcom.com/us/en/vmware-tanzu/platform/tanzu-platform-for-cloud-foundry/10-0/tpcf/metric-registrar-index.html)
+service so that application metrics can be viewed in any tool that is able to consume those metrics from
+the [Cloud Foundry Loggregator](https://github.com/cloudfoundry/loggregator-release).
+Several tools exist that can do this,
+including [App Metrics for VMware Tanzu](https://techdocs.broadcom.com/us/en/vmware-tanzu/platform-services/app-metrics-for-tanzu/2-2/app-metrics/index.html).
+
+> [!NOTE]  
+> While this application is intended to be used with [ActuatorApi](../ActuatorApi/), it _can_ be used on its own if you
+> are looking for a simpler actuator or metrics example.
+
+## General pre-requisites
+
+1. Installed .NET 8 SDK
+1. Optional:
+    * [VMware Tanzu Platform for Cloud Foundry](https://techdocs.broadcom.com/us/en/vmware-tanzu/platform/tanzu-platform-for-cloud-foundry/10-0/tpcf/concepts-overview.html)
+    * [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html)
+    * [Metrics Registrar for VMware Tanzu Application Service](https://techdocs.broadcom.com/us/en/vmware-tanzu/platform/tanzu-platform-for-cloud-foundry/10-0/tpcf/metric-registrar-index.html)
+    * [App Metrics for VMware Tanzu](https://techdocs.broadcom.com/us/en/vmware-tanzu/platform/tanzu-platform-for-cloud-foundry/10-0/tpcf/metric-registrar-index.html)
```

Comment:
Both links have the same URL, is that intentional?

Reply:
Nope, fixing the App Metrics link